metabase/metabase
# Add helper text to database form fields

Epic: https://github.com/metabase/metabase/issues/19099

- Adds help tooltips for some of the database form fields. 
- Changes copy for the fields.

<img width="1007" alt="Screen Shot 2021-11-25 at 10 35 57 pm" src="https://user-images.githubusercontent.com/8542534/143493481-004a1646-a4b9-4d01-9a09-a7f86526dfdd.png">

How to test:
- Go to Admin > Databases > Create
- Choose different database type and check help tooltips
